Gemfile

```diff
@@ -119,6 +119,9 @@ group :test do
   # Used to mock environment variables
   gem 'climate_control', '~> 0.2'
 
+  # Used in specs with before context setups
+  gem 'database_cleaner-active_record'
```

Comment:
May be obvious from this comment and how its used in file, but the issue here is that the `before(:context)` (and after) don't have the same run-in-transaction behavior that we relied on before, so we need to explicitly handle cleanup so we don't leak data to other examples.

Reply:
FYI, I started using DatabaseCleaner as well for my end-to-end testing implementation: https://github.com/mastodon/mastodon/pull/25461